src/HonzaBotner/HonzaBotner.csproj

```diff
@@ -8,6 +8,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Chronic.Core" Version="0.4.0" />
+    <PackageReference Include="Hangfire.AspNetCore" Version="1.7.19" />
+    <PackageReference Include="Hangfire.MemoryStorage" Version="1.7.0" />
+    <PackageReference Include="Hangfire.PostgreSql" Version="1.8.2" />
```

Comment:
Not sure what was the reason, but @ostorc wanted to implement something else (with simpler API for us probably), see #102.